docs/serverless/security-spaces.mdx

```diff
@@ -0,0 +1,16 @@
+---
+slug: /serverless/security/security-spaces
+title: Spaces and ((elastic-sec))
+description: Learn how spaces work in ((elastic-sec)).
+tags: [ 'serverless', 'security', 'reference' ]
+---
+
+((elastic-sec)) supports the organization of your security operations into logical instances with the <DocLink slug="/serverless/spaces">spaces</DocLink> feature. Each space in ((kib)) represents a separate logical instance of ((elastic-sec)) in which detection rules, rule exceptions, value lists, alerts, Timelines, cases, and ((kib)) advanced settings are private to the space and accessible only by users that have role privileges to access the space. For details about configuring privileges for ((es)) and ((kib)), refer to <DocLink slug="/serverless/security/detections-requirements" section="detections-permissions-section" />.
```

Comment:
Bit confused why we're linking to detections requirements when the sentence is talking about ES and Kib.

Reply:
Would it make sense to link [here](https://elastic-dot-co-docs-production-fr7hwqewk-elastic-dev.vercel.app/current/serverless/general/assign-user-roles) instead?

Reply:
Hm, good question. This page mostly mirrors the [stateful version](https://www.elastic.co/guide/en/security/master/security-spaces.html), which also points to the detections requirements, so this was just going for parity. But it does seem a little weird to focus on _just_ detections requirements, when several other features also have space-bound privilege requirements.

I think something like [this general requirements page](https://www.elastic.co/docs/current/serverless/security/requirements-overview) might be a better link here, since it gives info on space & index privs AND points to other feature-specific req pages. I'll whip up a new commit that makes this change to both serverless and stateful...